.github/workflows/e2e_tests.yaml

```diff
@@ -98,7 +98,7 @@ jobs:
           flutter test integration_test/e2e_test.dart --dart-define=CI=true
 
   ios:
-    runs-on: macos-13
+    runs-on: macos-12
```

Comment:
Maybe we should add a todo to go back to macos-13 or explain why we went back to 12?